carto/datasets.py

```diff
@@ -190,7 +190,8 @@ def create(self, archive, interval=None, **import_args):
 
         import_job = manager.create(archive) if interval is None \
             else manager.create(archive, interval)
-        import_job.run(**import_args)
+        if interval is None:
```

Comment:
We can move up the if to avoid the if/else of the previous line

Reply:
you're right! Changing it